### PR TITLE
docs: add integrations page

### PR DESCRIPTION
**Pull request by migmartri**
image

### CODE REVIEW

File: docs/integrations/integrations.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+sidebar_position: 1
+---
+
+# Chainloop Integrations
+
+Operators can extend Chainloop functionality by setting up third-party integrations that operate on your attestation metadata. Integrations can range from sending a Slack message, uploading the attestation to a storage backend or sending a Software Bill Of Materials (SBOMs) to a third-party service for analysis, for example.
+
+![FanOut Plugin](../img/fanout.png)
+
+Below you can find the list of currently available integrations. If you can't find the integration you are looking for, feel free [to reach out](https://github.com/chainloop-dev/chainloop/issues) or [contribute your own](https://github.com/chainloop-dev/chainloop/blob/main/app/controlplane/plugins/README.md)!
```
Comment (danlishka) [inline docs/integrations/integrations.md:13]: I like it!

File: docs/integrations/integrations.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+├──────────────────┼─────────┼──────────────────────┼───────────────────────────────────────────────────────────┤
+│ oci-registry     │ 1.0     │                      │ Send attestations to a compatible OCI registry            │
+├──────────────────┼─────────┼──────────────────────┼───────────────────────────────────────────────────────────┤
+│ discord-webhook  │ 1.1     │                      │ Send attestations to Discord                              │
+└──────────────────┴─────────┴──────────────────────┴───────────────────────────────────────────────────────────┘
+```
+
+Once you find your integration, i.e `oci-registry`, it's time to configure it.
+
+Configuring a integration has two steps: 1) register it in your organization and 2) attach the registered integration to your workflows.
```
Comment (danlishka) [inline docs/integrations/integrations.md:45]: an integration

File: docs/integrations/integrations.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    --opt repository=[region]-docker.pkg.dev/[my-project]/[my-repository] \
+    --opt username=_json_key \
+    --opt "password=$(cat service-account.json)"
+```
+
+> Note: You can find more examples on how to configure this specific integration [here](https://github.com/chainloop-dev/chainloop/tree/main/app/controlplane/plugins/core/oci-registry/v1)
+
+### Attaching an integration
+
+Once the integration is registered, we can attach it to any workflow. In practice this means that attestations and material information received by this workflow will be sent to the registered integration.
```
Comment (danlishka) [inline docs/integrations/integrations.md:92]: received by the workflow or generated in the workflow

**Review by danlishka — APPROVED** [review]
Please fix a typo and we are good to go.

File: docs/integrations/integrations.md (hunk 3 above)
Comment (migmartri) [reply docs/integrations/integrations.md:92]: good point, I like the latter.

I´ĺl update in both places

Thanks

**Pull request merged**